osu.Game/Overlays/News/Sidebar/MonthPanel.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Graphics.Containers;
+using osuTK;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics;
+using System.Linq;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Sprites;
+
+namespace osu.Game.Overlays.News.Sidebar
+{
+    public class MonthPanel : CompositeDrawable
+    {
+        public readonly BindableBool IsOpen = new BindableBool();
+
+        private readonly FillFlowContainer postsFlow;
+
+        public MonthPanel(List<APINewsPost> posts)
+        {
+            Width = 160;
```

Comment:
Still not sure though. Each tab has a defined size (33.75x15) with 5px spacing between them and the whole container have 5px padding, so total width will be 160px anyway. Hardcoded tab size is easier to maintain because we can just use `FillFlowContainer` instead of using `GridContainer` to make tabs perfectly-centered if we will use relative sizing